README.md

```diff
@@ -36,15 +68,15 @@ dependencies {
 }
 ```
 
-```kotlin
-implementation("io.projectliberty:siwf:0.0.1")
-```
+## 🛠 **Usage For Your App**
+When you decide to use the SIWF SDK in your own app, follow the steps below for easy integration:
 
-## **Usage**
-When you decide to use the SIWF SDK in your own app, follow the steps before for easy integration:
+### **1️⃣ Define an `authRequest`**
+- Refer to the Demo App for examples of encoded and non-encoded requests.
```

Comment:
@wilwade Only thought here is to define what an authRequest is. Do you have a recommendation on copy? Also, I thought we changed these all over to `authorizationRequest`... maybe that was `authorizationCode`. Any notes on changing or keeping the request name?

Reply:
I am ok with `authRequest` (we do that everywhere still) as it can mean authentication or authorization when it comes to the request side.

What about just `Define the SIWF Authentication Details` as the header?